error_reporting/google/cloud/error_reporting/__init__.py

```diff
@@ -14,5 +14,32 @@
 
 """Client library for Stackdriver Error Reporting"""
 
+from google.cloud.error_reporting.client import Client, HTTPContext
```

Comment:
No commas in imports:

```python
from google.cloud.error_reporting.client import Client
from google.cloud.error_reporting.client import HTTPContext
```

https://google.github.io/styleguide/pyguide.html#Imports_formatting